metadata_layer.py

```diff
@@ -202,14 +203,24 @@ def apply(self, destination, replace=None):
 
         made_changes = False
 
-        if self.sort_name != destination.name:
-            destination.name = self.sort_name
+        if hasattr(destination, 'name'):
+            # For Contributor objects.
+            name_attr = 'name'
+        elif hasattr(destination, 'sort_name'):
+            # For ContributorData objects.
+            name_attr = 'sort_name'
+
```

Comment:
Now (or the near future) might be the time to fix this by renaming the database field from contributors.name to contributors.sort_name.